JoinColony/colonyDapp
# Add TokenSalesTable component

Added TokenSalesTable component to CoinMachine page

**New stuff** ✨

* Expanded props of TableHeader in order to assign custom classes to it without TS yelling at us.
* Added UI for TokenSalesTable component used on CoinMachine

## How to test
1. Just enter the CoinMachine page to see the empty state of the table
2. To check how the table looks with data you could copy-paste this on the table component:

```
const placeholderData = [
    {
      saleEnd: '21/05 18:00',
      amount: 'SOLD OUT',
      price: 0.0009,
      priceStatus: TokenPriceStatuses.PRICE_SOLD_OUT,
    },
    {
      saleEnd: '21/05 17:00',
      amount: 'SOLD OUT',
      price: 0.0008,
      priceStatus: TokenPriceStatuses.PRICE_SOLD_OUT,
    },
    {
      saleEnd: '21/05 16:00',
      amount: '2500/4500',
      price: 0.0007,
      priceStatus: TokenPriceStatuses.PRICE_UP,
    },
  ];
```
And replace where `tableData` is used with `placeholderData`



![FireShot Capture 398 - Colony - localhost](https://user-images.githubusercontent.com/18473896/126715993-fb508bc0-fe96-4140-889d-266f90e85501.png)

![FireShot Capture 399 - Colony - localhost](https://user-images.githubusercontent.com/18473896/126716597-1f0ae1ec-704e-47a7-a31d-a1a094cc7dea.png)


Resolves DEV-455
